### ISSUE TITLE
Synchronous loading strategy

### ISSUE THREAD

**Issue by rektide** [derangeddk/serialize-every-error#4]
Can we make a synchronous loading strategy? The async makes it quite hard for us to wedge this into our apps, which tend to have fairly synchronous start-up paths.

**Comment by hypesystem** [comment]
Yes! I have some ideas for this, and will prioritize adding them.

The asynchronous loader exists to minimize the number of files loaded into memory to the ones actually needed. I'm thinking of exposing `serialize-every-error/base` and `serialize-every-error/all` as synchronous exports that come pre-packaged. Would that fit your needs?

**Comment by hypesystem** [comment]
I have published the base serializer synchronous export in version 0.1.4, let me know if that helps somewhat :smile:
